pkg/detectors/databrickstoken/databrickstoken.go

```diff
@@ -39,25 +41,23 @@ func (s Scanner) FromData(ctx context.Context, verify bool, data []byte) (result
 	domainMatches := domain.FindAllStringSubmatch(dataStr, -1)
 
 	for _, match := range matches {
-		if len(match) != 2 {
-			continue
-		}
 		resMatch := strings.TrimSpace(match[1])
 
 		for _, domainmatch := range domainMatches {
-			if len(domainmatch) != 2 {
-				continue
-			}
 			resDomainMatch := strings.TrimSpace(domainmatch[1])
 
 			s1 := detectors.Result{
 				DetectorType: detectorspb.DetectorType_DatabricksToken,
 				Raw:          []byte(resMatch),
-				RawV2:        []byte(resMatch + resDomainMatch),
+				RawV2:        []byte(fmt.Sprintf("%s:%s", resMatch, resDomainMatch)),
```

Comment:
@ankushgoel27 Can this be changed back to what it was?

Reply:
@zricethezav yes, i can change it to the way it was but i thought this is much better as having `:` makes it more readable.

Reply:
@ankushgoel27 please change it back.

Reply:
@zricethezav done